lib/dialects/mysql/query-generator.js

```diff
@@ -253,6 +253,8 @@ module.exports = (function() {
     },
 
     attributeToSQL: function(attribute) {
+      attribute = Utils.formatReferences(attribute);
```

Comment:
Please move this in the if clause after [line 296] (https://github.com/sequelize/sequelize/pull/3747/files#diff-9a80fbfe477740c76400d67d3565d303R296) so it's not called if not necessary. This applies to every of these calls.